src/main/pages/workspace-admin/servers.adoc

```diff
@@ -83,10 +94,13 @@ Server may require to be exposed publicly but access should be restricted only f
 }
 ----
 
-Where:
+The following describes the attributes:
 
-* secure - is an attribute that indicates whether server should be exposed as secure one. False value is a default one;
-* unsecuredPaths - is an attribute for configuring secure servers and it contains an comma-separated list of URI-s which are considered as non-secure on the given server and can be accessible without token. It may be needed when server provide any public APIs, like API endpoint for health checks;
-* cookiesAuthEnabled - is an attribute that indicates whether token should be searched in cookies. By default it is disabled. And server developer may enable it if he is sure that his servers can’t be attacked by CSRF or has special protection from it.
+* `secure` - indicates whether the server is exposed as secure. The default value is `False`.
+* `unsecuredPaths` - configures the secure servers. It contains a comma-separated list of URLs which are considered non-secure on the given server and can be accessible without the token. It may be needed when the server provides any public APIs. The API endpoint for health checks is an example.
```

Comment:
```suggestion
`unsecuredPaths`:: Configures the secure servers. It contains a comma-separated list of URLs that are considered non-secure on the given server and can be accessible without a token. It may be needed when the server provides any public APIs. The API endpoint for health checks is an example.
```